db/clinic/routers/schedule.py

```
@@ -23,6 +33,14 @@ def create_schedule(request: schemas.ScheduleBase, db: Session = Depends(get_db)
 def update_schedule(id: int, request: schemas.ScheduleBase, db: Session = Depends(get_db)):
     return schedule.update_schedule(id, request, db)
 
+@router.put("/{id}/book", response_model=schemas.ScheduleShow)
+def book_schedule(id: int, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)